contracts/governance/Staking/Staking.sol

```diff
@@ -373,6 +477,46 @@ contract Staking is
         emit StakingWithdrawn(msg.sender, amount, until, receiver, isGovernance);
     }
 
+    /**
+     * @notice Send user' staked tokens to a receiver.
+     * This function is dedicated only for governance vesting withdrawal.
+     *
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
+     * @param receiver The receiver of the tokens. If not specified, send to the msg.sender
+     * @param vesting The vesting address which act as the caller here.
+     * or just unlocked tokens (false).
+     * */
+    function _withdrawDirect(
```

Comment:
it should limit withdrawal by only team vesting contracts
this implementation allows to withdraw tokens from any vesting contract
it used to be controlled by diff in Vesting and TeamVesting proxy contracts where governanceWithdrawTokens was overridden to revert, so there was no way to call it from Vesting contract.
here you can use VestingRegistry.vestingContracts[tokenOwner][type] where type == uint256(VestingType.TeamVesting) to check that vesting is TeamVesting